gcli/src/cmd/send.rs

```diff
@@ -58,30 +53,33 @@ pub struct Send {
 }
 
 impl Send {
-    pub async fn exec(&self, signer: Signer) -> Result<()> {
-        let tx = signer
-            .calls
-            .send_message(
+    pub async fn exec(&self, app: &impl App) -> Result<()> {
+        let signer = app.signer().await?;
+        let gas_limit = if self.gas_limit == 0 {
```

Comment:
what if user truly wants to send message with 0 gas limit?
for example, it's valid case for reply nowadays

Reply:
I wouldn't do that

Reply:
https://github.com/gear-tech/gear/issues/3709